backend/app/assets/javascripts/spree/backend/admin.js

```diff
@@ -53,24 +53,48 @@ $.fn.radioControlsVisibilityOfElement = function(dependentElementSelector){
 }
 
 var handle_date_picker_fields = function(){
-  $('.datepicker').datepicker({
-    dateFormat: Spree.t('date_picker.js_format', { default: 'yy/mm/dd' }),
-    dayNames: Spree.t('abbr_day_names'),
-    dayNamesMin: Spree.t('abbr_day_names'),
-    firstDay: parseInt(Spree.t('date_picker.first_day', { default: '0' }), 10),
-    monthNames: Spree.t('month_names'),
-    prevText: Spree.t('previous'),
-    nextText: Spree.t('next'),
-    showOn: "focus"
+  $('.datepicker').flatpickr({
```

Comment:
since we are making changes here, what about moving all the datepicker code to a separate file?